workshop.wwwapi/Data/DatabaseContext.cs

```diff
@@ -8,20 +9,43 @@ namespace workshop.wwwapi.Data
     public class DatabaseContext : DbContext
     {
         private string _connectionString;
+
+        private DateTime _createdDateTime;
         public DatabaseContext(DbContextOptions<DatabaseContext> options) : base(options)
         {
-            var configuration = new ConfigurationBuilder().AddJsonFile("appsettings.json").Build();
+            //Loading the defaultconnectionstring value from the appsettings.json
+            var configuration = new ConfigurationBuilder().AddJsonFile("appsettings.Example.json").Build();
```

Comment:
why are you loading from the EXAMPLE appsettings; if you write any secrets in taht file they will be committed to github, the idea is to duplicate that file as `appsettings.json` and that .gitignore is saying htat `appsettings.json` is to be ignore and not committed to the repo

Reply:
I forgot this as I first wrote the password etc in the EXAMPLE file, but I created another file where i put the passwords, as I realized that my passwords were about to be pushed, I forgot to change this as well

Reply:
resolved